wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConfigImpl.java

```
@@ -96,6 +97,11 @@ private void addConfigValues(String section, String subsection, Set<String> name
     public Config set(String name, String value) throws GitException {
         ConfigKey key = parseName(name);
         repository.getConfig().setString(key.section, key.subsection, key.name, value);
+        try {
+            this.repository.getConfig().save();
+        } catch (IOException e) {
+            throw new GitException("Failed to save to config file");
```

as exception is dropped, should we at least log it ?